inputs/worker_inner.json

```diff
@@ -0,0 +1,158 @@
+{
+  "client_version": "{{VERSION}}",
+  "prebuild_plugins": [
+    {
+      "name": "add_filesystem",
+      "args": {
+        "koji_hub": "{{KOJI_HUB}}",
+        "from_task_id": "{{FILESYSTEM_KOJI_TASK_ID}}"
+      }
+    },
+    {
+      "name": "select_platform",
```

Comment:
worker will already know which platform from orchestrator right?
so that might be actually plugin for orchestrator?